toolz/curried/__init__.py

```diff
@@ -23,32 +23,39 @@
 See Also:
     toolz.functoolz.curry
 """
+import inspect
 
+from .operator import *
```

Comment:
Do we really want all of `operator` in the `curried` namespace?  I think it's reasonable to need to get them from `toolz.curried.operator`.  Perhaps change this to `from . import operator`.